### DISCUSSION TITLE
Pipeline technical meetings moved to September 7 and 8

### DISCUSSION THREAD

**Discussion by avneeshsingh** [daisy/pipeline#695] [Meetings]
Dear all,

The next set of Pipeline technical meetings were scheduled on August 10 and 11, 2023. But this is holidays time in parts of the world, and we are expecting low participation.
Therefore, we are moving the meetings to September 7 and 8, 2023.

With regards
Avneesh Singh

**Comment by bertfrees** [comment]
On the agenda for the meetings are the following proposed topics:

- The new Pipeline charter: We are currently writing the charter of the new year (2023-2024). We are using the responses of the Pipeline usage survey. Some of the responses require some more elaboration or discussion. We therefore invite you to explain your responses in some more detail during the meeting.
- The [DAISY Technical Meeting](https://daisy.org/technical-meeting-2023/) in London: We would like to know who of you will participate in the upcoming DAISY Technical meeting held on November 10th in London, in which DAISY Pipeline will also be covered. In addition, we are considering organising a technical workshop for developers on the day before, during the Accessible Publishing Conference. We would like to know who would be interested in such a workshop and which topic(s) you would suggest.

[This is the Zoom link](https://us02web.zoom.us/j/84980183752?pwd=TkRvUTJLbG8zbVhXdE9HQXlVc3RCUT09).
